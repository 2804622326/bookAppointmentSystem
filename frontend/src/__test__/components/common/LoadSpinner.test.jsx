import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import LoadSpinner from '../../../components/common/LoadSpinner.jsx';

test('renders spinner with variant', () => {
  const { asFragment } = render(<LoadSpinner variant="primary" />);
  expect(asFragment()).toMatchSnapshot();
});
