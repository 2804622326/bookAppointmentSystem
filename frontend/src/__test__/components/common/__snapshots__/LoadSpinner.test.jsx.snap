// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders spinner with variant 1`] = `
<DocumentFragment>
  <div
    class="d-flex justify-content-center align-items-center mt-5"
    style="height: 100%;"
  >
    <div
      class="spinner-border text-primary"
    />
  </div>
</DocumentFragment>
`;
